molecule/docker-prod/converge.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Converge
+  hosts: ome-dockr-prod1.openmicroscopy.org
+  tasks:
+    - name: Workaround to get host IP inside docker
+      shell: hostname -I | cut -d' ' -f1
+      register: hostname_ip
+      check_mode: false
+      changed_when: false
+      tags:
+        # Ignore [306] Shells that use pipes should set the pipefail option
```

Comment:
But _shouldn't_ we be setting `pipefail`? ;)

Reply:
Could do, but I'd need to find out how, and in any case since the incorrect output would fail in the playbook